packages/koa-shopify-graphql-proxy/README.md

```diff
@@ -85,3 +85,27 @@ app.use(
   }),
 );
 ```
+
+### Custom Headers
+
+If you need to pass headers to the proxied request, you can than add those headers during the configuration of the proxy.
+
+```javascript
+// server/index.js
+import koa from 'koa';
+import session from 'koa-session';
+import proxy, {ApiVersion} from '@shopify/koa-shopify-graphql-proxy';
+
+const app = koa();
+
+app.use(session());
+
+app.use(() =>
+  proxy({
```

Comment:
Maybe give an example where we pull an existing request header from context and pass it in the proxy. For example `X-Request-ID`